extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -135,6 +136,16 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 		});
 	}
 
+	function extendTargetingForBlockedTraffic(adElement) {
+		if (instartLogic && instartLogic.isBlocking()) {
+			// TODO: for testing only!
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)